frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-design/experiment-design.component.ts

```diff
@@ -142,10 +142,10 @@ export class ExperimentDesignComponent implements OnInit, OnChanges, OnDestroy {
 
     if (this.isContextChanged) {
       this.isContextChanged = false;
-      this.partition.clear();
-      this.condition.clear();
-      this.partitionDataSource.next(this.partition.controls);
-      this.conditionDataSource.next(this.condition.controls);
+      this.partition?.clear();
```

Comment:
why are changes made to simple experiment form?

Reply:
components were loading before we choose which experiment in overview tab so we added this.

Reply:
right ok